democracy/views/section_comment.py

```diff
@@ -437,6 +444,7 @@ class CommentFilterSet(django_filters.rest_framework.FilterSet):
     created_at__lt = django_filters.IsoDateTimeFilter(field_name='created_at', lookup_expr='lt')
     created_at__gt = django_filters.rest_framework.IsoDateTimeFilter(field_name='created_at', lookup_expr='gt')
     comment = django_filters.ModelChoiceFilter(queryset=SectionComment.objects.everything())
+    section = django_filters.CharFilter(field_name='section__id')
```

Comment:
This concerns the Browsable API filter options. Removes "ModelChoiceFilter" and replaces it with a CharFilter so it's similar to the hearing filter that is on line 442